README.md

```diff
@@ -16,63 +39,191 @@
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
```

Comment:
Good job with the customization of the Readme. Let's make sure all sections on the Readme template are present in yours as the previous reviewers requested. Kindly add the "Feature Features" section below and don't forget to add at least one feature.